pkg/plugins/policies/meshtrafficpermission/api/v1alpha1/meshtrafficpermission.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package kuma.plugins.policies.meshtrafficpermission.v1alpha1;
+
+import "mesh/options.proto";
+option go_package = "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission/api/v1alpha1";
+
+import "common/v1alpha1/targetref.proto";
+import "config.proto";
+
+option (doc.config) = {
+  type : Policy,
+  name : "MeshTrafficPermission",
+  file_name : "meshtrafficpermission"
+};
+
+// MeshTrafficPermission defines permission for traffic between data planes
+// prxoies.
+message MeshTrafficPermission {
+  option (kuma.mesh.policy) = {
+    skip_registration : false,
+  };
+
+  // TargetRef is a reference to the resource the policy takes an effect on.
+  // The resource could be either a real store object or virtual resource
+  // defined inplace.
+  kuma.common.v1alpha1.TargetRef targetRef = 1;
+
+  message From {
+    // TargetRef is a reference to the resource that represents a group of
+    // clients.
+    kuma.common.v1alpha1.TargetRef targetRef = 1;
+
+    message Default {
```

Comment:
but override will have the same values, right? I think this message should be named in another way.

Reply:
I see, we can name the message `conf`